### DISCUSSION TITLE
will this work on rhel 8?

### DISCUSSION THREAD

**Discussion by polarmutex** [pop-os/shell#1574] [General]
the version of gnome is 3.32 on rhel 8.7, the readme says 3.36 is required. just checking if 3.36 was a hard requirement or not just tested

**Comment by jacobgkau** [comment]
Ubuntu 18.04 (from April of 2018) shipped GNOME Shell version 3.28, and the extension is [explicitly unsupported](https://github.com/pop-os/shell/tree/master) on a version that old. The [master_jammy](https://github.com/pop-os/shell/tree/master_jammy) branch (for GNOME 42) was split off from [master](https://github.com/pop-os/shell/tree/master) (for 3.36 through 40) because of significant differences in how GNOME Shell and Mutter work between versions (and from what I recall, the `master` branch supporting multiple versions had various places where the code path had to diverge depending on the GNOME Shell version.)

I would not expect the extension to work on version 3.32 without making changes to the code.

**Reply by jacobgkau** [reply]
Searching past issues, it sounds like some users on 3.34 had to make code changes to get it working on that version: https://github.com/pop-os/shell/issues/141 (so those changes, and potentially more, would be required for 3.32.)
